.github/workflows/pr-sanity.yaml

```diff
@@ -0,0 +1,47 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Sanity Checks
+
+# Controls when the action will run.
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
what about cherry-pick pull requests on release branches?

Reply:
I almost sure that travis is not running there